core/src/mindustry/world/blocks/units/RepairPoint.java

```diff
@@ -174,7 +174,6 @@ public void updateTile(){
                 multiplier = liq.valid(this) ? 1f + liquids.current().heatCapacity * coolantMultiplier : 1f;
             }
 
-            target.damage(1);
```

Comment:
this is needed so the repair beam effect is replicated client side
the unit later gets healed back

Reply:
( do not remove )

Reply:
refering to the "repair points give damage boost instead of heal" feature